spread.yaml

```diff
@@ -286,6 +287,14 @@ suites:
      # shellcheck source=/dev/null
      source "$HOME/venv/snapcraft/bin/activate"
      pip install --upgrade pip wheel setuptools
+
+     # Install python-apt using pip 20.1, last known working version.
```

Comment:
Instead of this you can try and apt install and use --system-site-packages for pyvenv